### PR TITLE
Add recipe for NumberNine CMS 0.1

### PR DESCRIPTION
**Pull request by williarin**
| Q             | A
| ------------- | ---
| License       | MIT

Recipe for [NumberNine CMS](https://github.com/numberninecms/cms)

### CODE REVIEW

File: numberninecms/cms/0.1/manifest.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "bundles": {
+        "NumberNine\\Bundle\\NumberNineBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "public/": "%PUBLIC_DIR%/",
+        "docker/": "./docker/"
```
Comment (Nyholm) [inline numberninecms/cms/0.1/manifest.json:8]: Let's use "copy-from-package" for the public and docker folders. That will make it easier for you to keep them updated.

**Comment by Nyholm** [comment]
Thank you. Im happy with this PR. 

Can you rebase it and make sure the CI is green?

**Comment by williarin** [comment]
I think I messed it up 🙁

As the package 0.1.* will not be used anymore and isn't compatible with Symfony 5.3 (failing tests), I upgraded the recipe by renaming the version folder "0.2" in the hope that would fix the problem.

Now the workflow still wants to `composer req "numberninecms/cms:^0.1"` instead of 0.2.
I'm not sure what I can do about it, any suggestion?

The recipe works fine when I require ^0.2.

**Comment by Nyholm** [comment]
No worries. I dont think you've messed up. It might be something wrong with the checker. You just found an edge case =)

I'll be happy to merge this. Please test this after merge to make sure it works as expected.
